src/js/components/nft/nft-create-form.jsx

```diff
@@ -135,6 +138,9 @@ class NftCreateForm extends Component {
             transaction_output_attribute['parent_transaction_id'] = this.state.txid;
         }
 
+        transaction_output_attribute.name = this.nft_name.value;
+        transaction_output_attribute.nft_description = this.nft_description.value;
```

Comment:
why do you use `name` and `nft_description`? `nft_` prefix seems to be extra in attribute name. what is the name pattern on other attributes?